lib/citybuilder/web/templates/post/_post_restore.html.eex

```diff
@@ -1,4 +1,32 @@
 <div class="post-restore hidden">
-  Post deleted. (°◇°) Click ~Restore Post~ to undelete... In 10 seconds your post will be be gone forever.
-  <%= link "Restore Post", to: post_path(@conn, :restore, @post), "ujs-method": "post", "ujs-remote": true, "ujs-eval": true, class: "btn btn-default post-restore-btn" %>
+<div class="card-top learn">
```

Comment:
Yet another tech debt. Copied this from `_post.html.eex`. Probably should do merge of these two templates in future.